web/src/constants.ts

```diff
@@ -93,3 +94,9 @@ export const PROFILE_EDIT_VIEW_NAMES = {
 export const RESPONSE_STATUS_CODE = {
   UNAUTHORIZED: 401,
 };
+
+export const MINIMUM_TECHNICAL_LEADS = 1;
```

Comment:
Enums in typescript might be a good way to deal with some contestants:
https://www.typescriptlang.org/docs/handbook/enums.html